### DISCUSSION TITLE
Diary-14-我要下象棋！ | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#330] [Announcements]
# Diary-14-我要下象棋！ | Zi-Zi's Journey

2024.04.08-2024.04.21

https://gz-metal-cell.github.io/2024/04/20/Diary-14-%E6%88%91%E8%A6%81%E4%B8%8B%E8%B1%A1%E6%A3%8B%EF%BC%81/

**Comment by promefire** [comment]
不对劲啊为什么Github不给我推送Metal哥更新的消息🤔

**Reply by GZ-Metal-Cell** [reply]
😭我没有像牛逼的 pro 哥那样开了 ImgBotApp😭那玩意每优化一次图片就会顺带提醒我一次该去骚扰 pro 神了😭
